addons/portal/controllers/portal.py

```diff
@@ -717,9 +722,15 @@ def _validate_address_values(
         # Complete the set of required fields based on the address type.
         country_id = address_values.get('country_id')
         country = request.env['res.country'].browse(country_id)
-        if address_type == 'delivery' or use_delivery_as_billing:
+        if (
+            not skip_address_required_fields
```

Comment:
IMHO, `skip_address_required_fields` should only bypass the results of `_get_mandatory_address_fields`, not the invoicing fields requirements.

We probably should also use/consider whether the view `address_b2b` is enabled, speaking with BOJE about it.